std/format/package.d

```diff
@@ -198,66 +198,81 @@ contain exactly one $(I format specifier), which specifies the way,
 the elements of the compound type will be formatted. This $(I format
 specifier) can be a $(I compound indicator) itself.
 
+$(SECTION4 Flags)
 
+There are several flags, that affect the outcome of the formatting.
 
+$(BOOKTABLE ,
+   $(TR $(TH Flag) $(TH Semantics))
+   $(TR $(TD $(B '-'))
+        $(TD When the formatted result is shorter then the value
+             given by the width parameter, normally, the output is
+             right justified. With the $(B '-') flag this is changed
+             to left justification.
+
+             There are two exceptions where the $(B '-') flag has a
+             different meaning: With $(B 'r') it denotes to use little
+             endian and in case of a compound indicator it means that
+             no special handling of the members is applied.))
+   $(TR $(TD $(B '+')&nbsp;/&nbsp;$(B '&nbsp;'))
+        $(TD Applies only to numbers. Normally, positiv numbers are not
+             preceded by anything. With one of these two flags present,
+             positiv numbers are preceded by a plus sign or a space.
+             When both flags are present, a plus sign is used.
+
+             In case of $(B 'r') it denotes to use big endian.))
+   $(TR $(TD $(B '0'))
+        $(TD Applies only to numbers that are printed right justified.
+             If the zero flag is present, the space left to the number is
+             filled with zeros instead of spaces.))
+   $(TR $(TD $(B '#'))
+        $(TD An alternativ output shall be used. This depends on the type
+             to be formatted and the $(I format character) used. See the
+             sections below for more information.))
+)
 
-    $(BOOKTABLE Flags affect formatting depending on the specifier as
-    follows., $(TR $(TH Flag) $(TH Types&nbsp;affected) $(TH Semantics))
+$(SECTION4 Width\, Precision and Separator)
 
-    $(TR $(TD $(B '-')) $(TD numeric, bool, null, char, string, enum, pointer) $(TD Left justify the result in
-        the field.  It overrides any $(B 0) flag.))
+The $(I width) specifies the minimum width of the result.
```

Comment:
```suggestion
The $(I width) parameter specifies the minimum width of the result.
```

Reply:
This sentence is kind of tautological: "The width specifies the width of the result".

Reply:
The important thing is _minimum_.